.circleci/config.yml

```diff
@@ -156,6 +156,7 @@ workflows:
           pre-steps:
             - font_awesome
       - cypress/run:
+          command: npm start & cypress run --config
```

Comment:
I'm not convinced this is going to work.  I don't really know why we're seeing coverage errors on these tests, but here's the error specifically: https://dashboard.cypress.io/projects/ep9zr6/runs/2234/test-results/5c7a4b7b-c872-4496-a645-94d24c06ef96?statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D

I think we need to prove the fix works before we change these by probably temporarily modifying the main action or adding a temporary new one.  Maybe remove this change from this PR and a separate PR can come out where we can make sure that the coverage reports don't run on CI?  It will probably take forcing an error to prove that no after each is called when tests are run on circleci.

Reply:
That's fair. I thought it would be a much more minor change. I'll put up a separate PR